src/components/Chakra/Grid.stories.mdx

```diff
@@ -0,0 +1,85 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+import { Box } from "@chakra-ui/react";
+import { Grid, GridItem } from "@chakra-ui/react";
+
+import { getCategory } from "../../utils/componentCategories";
+import VideoPlayer from "../VideoPlayer/VideoPlayer";
```

Comment:
I think these were accidentally added -- they're not used in this file.

Reply:
Ugh.  Leftover from the file I initially copied.  Now removed.